### PR TITLE
DEVICE: zyxelnwa, add storage, mempools and wireless

### PR DESCRIPTION
**Pull request by tomarch**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/polling/os/zyxelnwa.inc.php [hunk 1]
```diff
@@ -1,3 +1,6 @@
 <?php
 
 $hardware = $poll_device['sysDescr'];
+$version = explode("ITS", trim(snmp_get($device, '.1.3.6.1.4.1.890.1.15.3.1.6.0', '-Osqv'), '"'), 2);
```
Comment (laf) [inline includes/polling/os/zyxelnwa.inc.php]: can you not just do `list($version,) = explode....` here?

File: includes/polling/os/zyxelnwa.inc.php [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+$version = $version[0];
+$serial = trim(snmp_get($device, '.1.3.6.1.4.1.890.1.15.3.1.12.0', '-Oqv'), '"');
```
Comment (laf) [inline includes/polling/os/zyxelnwa.inc.php]: Do you not have the MIB to use here? You can then do an snmp multi get.

**Comment by laf** [comment]
Thanks @tomarch. the only last thing, sorry to be a pain is to rename the MIBs. The filenames should be that of the actual MIB name to make it easier to find. I.e: `mibs/zyxelnwa/ZYXEL-ES-COMMON.MIB` should be `mibs/zyxelnwa/ZYXEL-ES-COMMON`. The name is from the line: `ZYXEL-ES-COMMON DEFINITIONS ::= BEGIN` (line 13) of the mib.

**Comment by tomarch** [comment]
No problem ;)

**Comment by laf** [comment]
image

**Pull request merged**